### DISCUSSION TITLE
Control GPIO Grounds With HASPOne?

### DISCUSSION THREAD

**Discussion by FlapjackStaxx** [HASwitchPlate/HASPone#189] [Q&A]
I'm trying to control some 12v LEDs and just needing to switch grounds to each color on the strip. Each color has its own pin and mosfet. I can get this to work with ESPHome but that system was garbage.

Is there a way to control which GPIO pins see ground with say a button press?
I'm on a NODEMCU 8266 currently

**Comment by aderusha** [comment]
Currently this project does not really offer generalized control of GPIO.  It's not on the roadmap, but is a maybe for future releases.  The topic keeps coming up which suggests I should probably look further into it...
